openhtf/core/measurements.py

```diff
@@ -139,6 +140,19 @@ def _initialize_value(self):
     else:
       self.measured_value = MeasuredValue(self.name)
 
+  def __deepcopy__(self, memo):
```

Comment:
The measured_value check seems like it's still useful because of how measured_value is manipulated when setting dimensions in __setattr__. @fahhem's suggestion might be an easy fix, it makes sense that initializing a Measurement with a measured_value should be allowed.

```
  def __init__(self, name, **kwargs):
    measured_value = kwargs.pop('measured_value', None)
    super(Measurement, self).__init__(name, **kwargs)
    self._initialize_value()
    self.measured_value = measured_value
```